js/actions/MessageActions.coffee

```diff
@@ -15,12 +15,12 @@ Persistence = _persistence MessageActions: module.exports =
     Dispatcher.handleServerAction {messages,last,get,type:"messages-load"}
 
   listenStation: (station,date) ->
-    if not date then date = window.urb.util.toDate (
```

Comment:
Why move the type conversion?

Reply:
So that we have access to the Date object in MessagePersistence. We may want to subtract from the date there (in case our query result is 0 messages), so having the original Date object there is much more workable than the stringified date.